addons/point_of_sale/tests/common_setup_methods.py

```diff
@@ -132,7 +117,14 @@ def setup_product_combo_items(self):
         {
             "name": "Desks Combo",
             "combo_item_ids": [
-                (6, 0, [product_4_combo_item.id, product_5_combo_item.id])
```

Comment:
forgot to remove those two products above (or they came back on rebase, dunno), cf red runbot

Reply:
Fixed :) Also fixed a few other places where there still seemed to be issues.